packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -340,6 +340,7 @@ const CFDsListing = observer(() => {
                                         key={`trading_app_card_${existing_account.display_login}`}
                                         onAction={(e?: React.MouseEvent<HTMLButtonElement>) => {
                                             const button_name = e?.currentTarget?.name;
+                                            setProduct();
```

Comment:
if we are setting a product, why is it empty?

Reply:
We are reseting product to empty here. and `available_ctrader_accounts` and `available_dxtrade_accounts` doesn't have any product key

Reply:
We're currently only using product for `zero_spread` and `swap_free` accounts. Until then we just need to reset product upon clicking on those products so that we keep their functionalities as it is.